Lombiq.HelpfulExtensions.Tests.UI/Constants/GeneratedMigrationCodes.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+
+namespace Lombiq.HelpfulExtensions.Tests.UI.Constants;
+
+internal static class GeneratedMigrationCodes
+{
+    // Environment.NewLine is used for this to work regardless of the mismatch of line ending style between the code
+    // file, the platform where it was compiled, and where it was executed.
+    public static string Page =
+        "_contentDefinitionManager.AlterTypeDefinition(\"Page\", type => type" + Environment.NewLine +
```

Comment:
Why not use `string.Join(Environment.NewLine, new [] { ... })`?

Reply:
After all, science is not about why, but why not?